src/Symfony/Component/Serializer/Normalizer/BackedEnumNormalizer.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+use Symfony\Component\Serializer\Exception\NotNormalizableValueException;
+
+/**
+ * Normalizes a {@see \BackedEnum} enumeration to a string or an integer.
+ *
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+class BackedEnumNormalizer implements NormalizerInterface, DenormalizerInterface, CacheableSupportsMethodInterface
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @throws InvalidArgumentException
+     *
+     * @return string
+     */
+    public function normalize($object, string $format = null, array $context = [])
+    {
+        if (\PHP_VERSION_ID < 80100) {
+            throw new \LogicException('"BackedEnumNormalizer" cannot be used as the PHP version is lower than 8.1.');
+        }
+
+        if (!$object instanceof \BackedEnum) {
+            throw new InvalidArgumentException('The object must belong to a backed enumeration.');
+        }
+
+        return $object->value;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsNormalization($data, string $format = null)
+    {
+        return \PHP_VERSION_ID >= 80100 && is_subclass_of(\get_class($data), '\BackedEnum');
```

Comment:
`$data instanceof \BackedEnum` is better (and it works without breaking if `$data` is not an object)